digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/tierekisteri/TierekisteriClient.scala

```diff
@@ -43,62 +43,151 @@ object TRRoadSide {
 sealed trait TRTrafficSignType {
   def value: Int
   def trafficSignType: TrafficSignType
+  def source: Seq[String]
 }
 object TRTrafficSignType {
-  val values = Set(SpeedLimit, EndSpeedLimit, SpeedLimitZone, EndSpeedLimitZone, UrbanArea, EndUrbanArea, PedestrianCrossing, PedestrianCrossing, MaximumLength, Warning, NoLeftTurn, NoRightTurn, NoUTurn,
+  val values = Set(SpeedLimit, EndSpeedLimit, SpeedLimitZone, EndSpeedLimitZone, UrbanArea, EndUrbanArea, PedestrianCrossing, MaximumLength, Warning, NoLeftTurn, NoRightTurn, NoUTurn,

```

Comment:
Once the object **TRTrafficSignType** is used for CSVimport too, we could create a generic one

Reply:
Let's do that modification on next time we work with TR TrafficSign Types, only to avoid to test everything again.
I will add a TODO for that.